mscore/bendcanvas.cpp

```diff
@@ -1,8 +1,8 @@
 //=============================================================================
-//  MusE Score
-//  Linux Music Score Editor
+//  MuseScore
+//  Music Composition & Notation
 //
-//  Copyright (C) 2010-2011 Werner Schweer and others
+//  Copyright (C) 2010-2019 MuseScore BVBA
```

Comment:
Not sure about this copyright change

Reply:
I surely didn't mean that. `mscore/bendcanvas.cpp` is a file I created, git recognized it as a renamed file because half of the content is identical to one of the old files I deleted.

Reply:
Now changed copyright of all new files to `Werner Schweer and others` ;-)